doc/api/next_api_changes/2018-11-04-JMK.rst

```diff
@@ -0,0 +1,14 @@
+get_window_extents changes:
+---------------------------
+
+`.matplotlib.axes.Axes.get_window_extent` used to return a bounding box
+that was slightly larger than the axes, presumably to take into account
+the ticks that may be on a spine.  However, it was not scaling the tick sizes
+according to the dpi of the canvas, and it did not check if the ticks were
+visible, or on the spine.
+
+Now  `.matplotlib.axes.Axes.get_window_extent` just returns the axes extent
+with no padding for ticks.
+
+`.spines.get_window_extent` now takes into account ticks that are on the
+spine.
```

Comment:
Is a note needed for get_tightbbox changes?  (Haven't really followed whether it changed)

Reply:
Agreed, it changes if the ticks are outwards and there are no outwards tick labels.